src/components/NewsAndArticles/index.jsx

```diff
@@ -13,7 +13,7 @@ import { useSpring, animated } from 'react-spring';
 import useMedia from '../../Helpers/useMedia';
 import Article from '../Article';
 
-const ArticlesList = ({ articles, loading, error, animationDelay }) => {
+const ArticlesList = ({ data, loading, error, animationDelay }) => {
```

Comment:
why did you change the prop name? wasn't it better to pass data as articles prop and keep the component props as is?

Reply:
I changed it to follow the convention to send data to components in "data" name and in component i can name it whatever I want